docs/about/learning-resources.rst

```diff
@@ -6,7 +6,7 @@ Learning Resources
    :maxdepth: 2
    :hidden:
 
-   /newsletter
```

Comment:
We should continue listing the newsletter here, I think

Reply:
I can add it back in, but it's slightly weird as that's the "Newsletter and mailing lists" page. I think it'd make more sense to link to the newsletter archive here, but I'm not sure how to do that.

Reply:
Ah, it's still in the Stay Connected section -- ignore me :)